AGENTS.md

```
@@ -1,147 +1,266 @@
+# 生成 Changelog（交互式）
+npm run changelog
+
+# 清理构建产物
+npm run clean
 ```
 
-## 代码风格指南
+### 项目结构
 
-### 基本编码规范
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)